### ISSUE TITLE
Allow rationale to ignore directive

### ISSUE THREAD

**Issue by Kreyren** [hadolint/hadolint#441]
- [ ] This is a bug report
- [X] This is a feature request
- [X] I searched existing issues before opening this one

### Expected behavior

Expecting comments alike to be recognized for ignore directive:

```Dockerfile
# hadolint ignore=DL3007 # Latest is expected here
FROM gitpod/workspace-full:latest
```

### Actual behavior

![image](https://user-images.githubusercontent.com/11302521/85039780-5badb200-b188-11ea-9826-dbab1dd37c12.png)

**Comment by Kreyren** [comment]
> This is also not working which I think it should be. @SuperSandro2000 

FWIW i don't like doing this since it makes the code unnecessary longer

**Comment by SuperSandro2000** [comment]
You could swap the lines but done more flexibility would be nice.

**Comment by guillaume-d** [comment]
Duplicate of #370? #370 is more descriptive and ATM has more good input IMHO.

**Comment by Kreyren** [comment]
Confirming duplicate of https://github.com/hadolint/hadolint/issues/370
